oat-sa/tao-core
# Feat / ADF-436 / [GREP] - Usage - Update properties UI on selected resource

Jira https://oat-sa.atlassian.net/browse/ADF-436

In order to test:

- Create a class and go to Manage schema
- Add two properties (one will be primary and one will be secondary)
- Set Type to list - Single/Multiple choice - Search/Dropdown
- Set List values of primary to Parent list
- Set List values of secondary to child list
- Save changes and reload Manage Schema
- Set Depends on property of Secondary property to Primary
- Save changes and create item

Accepted criteria:

- You should be able to ONLY select secondary properties when a primary property has value. If not, secondary input should be disabled.
- If you have assigned values to both properties and then set primary to empty, secondary should be cleared and disabled.
- All changes should be persisted


If you want to directly test AC's without setting up the classes and properties you can access to class ADF-436 and item ADF-436 1 directly (which already has dependent properties defined)

TODO: _(Only for future reference purposes, can be ignored for PR review)_
- [x] Set proper primary URI on BE https://github.com/oat-sa/tao-core/pull/3103/files#diff-d663f726bfddec08e0b9a1cc9181ba9ecc8416358b2a1b2f0ad2f2721e063977R175
- [x] Change https://github.com/oat-sa/tao-core/pull/3103/files#diff-d663f726bfddec08e0b9a1cc9181ba9ecc8416358b2a1b2f0ad2f2721e063977R176 when BE is ready
- [x] Test secondary dropdown single choice reactivity (should clear when primary is cleared), fix if not working properly
- [x] (Optional) Refactor
- [ ] (Optional) Move styles and js to tao-core-ui-fe
